src/fileops.rs

```diff
@@ -6,27 +6,30 @@ use std::ffi::CString;
 use libc::c_int;
 use std::os::unix::io::{RawFd, AsRawFd};
 use std::ops::Drop;
+use std::io::Error;
+use std::error::Error as std_error;
```

Comment:
at least you can alias them. More than you can say for Java.

I rly enjoy my life when I've got guava.Optional and java.util.Optional, and you have to refer to one of them fully-qualified.